src/css/main.css

```diff
@@ -41,6 +41,12 @@ body {
       grid-template-columns: 2fr, 2fr, 2fr;
 }
 
+.survey-timeline-card {
+  border: 3px solid black;
+  padding: 15px;
+  margin-top: 10px;
```

Comment:
I think we should use `rem` instead of `px` for padding and margin property. 1 rem equals 16 px as default. It will be easy when transform responsive style.

Reply:
I made the change thank you for your advice.